testcases/crypto/aes_func.c

```diff
@@ -2030,6 +2031,8 @@ int main(int argc, char **argv)
     }
 
     testcase_setup(0);          //TODO
+
+    pkey = CK_FALSE;
```

Comment:
Do you ever set this to TRUE? Looks like you run it only once with pkey=FALSE....

Reply:
Yep ... below ...